packages/MorphicAPIExplorer-Core.package/MEXToolOverView.class/instance/initialize.st

```diff
@@ -3,6 +3,7 @@ initialize
 
 	super initialize.
 	self categories: self class categories.
+	self categories do:[:each| each methodList do: [:method | method category: each. method subMethods do:[:sub | sub category: each]]].
```

Comment:
Keine einheitliche Leerzeichensetzung, das vielleicht in eine deskriptive Methode auslagen